tests/testutils/ctx.go

```diff
@@ -174,7 +174,12 @@ func (ctx *RktRunCtx) RunGC() {
 		"gc",
 		"--grace-period=0s",
 	)
-	if err := exec.Command(ctx.rktBin(), rktArgs...).Run(); err != nil {
+
+	cmd := exec.Command(ctx.rktBin(), rktArgs...)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stdout
```

Comment:
You should be able to see the logs in the case of a test failure

On Wed, Apr 27, 2016 at 1:56 PM, JJ Łakis <notifications@github.com> wrote:

> In tests/testutils/ctx.go
> <https://github.com/coreos/rkt/pull/2513#discussion_r61244719>:
>
> > @@ -174,7 +174,12 @@ func (ctx *RktRunCtx) RunGC() {
> >  		"gc",
> >  		"--grace-period=0s",
> >  	)
> > -	if err := exec.Command(ctx.rktBin(), rktArgs...).Run(); err != nil {
> > +
> > +	cmd := exec.Command(ctx.rktBin(), rktArgs...)
> > +	cmd.Stdout = os.Stdout
> > +	cmd.Stderr = os.Stdout
>
> @iaguis <https://github.com/iaguis> My mistake. @jonboulle
> <https://github.com/jonboulle> Honestly, the motivation was semaphore CI
> where I can't see the logs. But logging it is also an idea.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/rkt/pull/2513/files/edfc480f9160b08d968ac02a1bdfdfaa8bdc04f1#r61244719>
>